src/main/java/mate/academy/model/Book.java

```diff
@@ -7,14 +7,16 @@
 import jakarta.persistence.Id;
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
-import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
 import org.hibernate.annotations.SQLDelete;
 import org.hibernate.annotations.Where;
 
-@Data
 @Entity
 @Table(name = "books")
 @SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
+@Getter
+@Setter
 @Where(clause = "is_deleted=false")
```

Comment:
```suggestion
@SQLRestriction(value = "is_deleted=false")
```

`@Where` is deprecated